### PR TITLE
Fix Codeowners for New Team Names

### PR DESCRIPTION
**Pull request by TimPansino**
Codeowners updated with new team names and all teams have been added back to the repository with write access.

### CODE REVIEW

**Comment by fallwith** [comment]
Was there an APM agent dues fee we missed that resulted in `ruby-agent` not being changed to `ruby` as well as part of this? 😄  We can contribute that change separately if need be.

**Comment by TimPansino** [comment]
> Was there an APM agent dues fee we missed that resulted in `ruby-agent` not being changed to `ruby` as well as part of this? 😄 We can contribute that change separately if need be.

It is changed to `ruby` in this PR already. Not sure what you're looking at exactly.

**Review by nrcventura — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -26,6 +26,6 @@
 /src/python/ @newrelic/python
 /tests/python/ @newrelic/python
 
-/.github/workflows/ruby.yml @newrelic/ruby-agent
-/src/ruby/ @newrelic/ruby-agent
-/tests/ruby/ @newrelic/ruby-agent
+/.github/workflows/ruby.yml @newrelic/ruby
+/src/ruby/ @newrelic/ruby
+/tests/ruby/ @newrelic/ruby
```
Comment (TimPansino) [inline .github/CODEOWNERS:31]: Ruby was changed here

**Comment by fallwith** [comment]
> > Was there an APM agent dues fee we missed that resulted in `ruby-agent` not being changed to `ruby` as well as part of this? 😄 We can contribute that change separately if need be.
> 
> It is changed to `ruby` in this PR already. Not sure what you're looking at exactly.

My apologies! I think I must have opened the `main` version of the file pre-PR-merge. Sorry about that, and thanks for squaring these changes away for all of us!

**Pull request merged**